definitions/types/gcp-security-iam.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "IAM",
+  "description": "IAM Roles And Conditions",
+  "additionalProperties": false,
+  "patternProperties": {
```

Comment:
Notice the lack of `iam`, if we go per-cloud we'd have to add that property here but then in the gcp-artifacts one couldn't configure it for another cloud.